src/combinatorics/additive/behrend.lean

```diff
@@ -0,0 +1,205 @@
+/-
+Copyright (c) 2022 Yaël Dillies, Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies, Bhavik Mehta
+-/
+import analysis.inner_product_space.pi_L2
+import combinatorics.additive.salem_spencer
+
+/-!
+# Behrend's bound on Roth numbers
+
+This file proves Behrend's lower bound on Roth numbers. This says that we can find a subset of
+`{1, ..., n}` of size `n / exp (O (log (sqrt n)))` which does not contain arithmetic progressions of
```

Comment:
The bound is probably not of this form, because if the `O` constant is `2` then you just get `1`...

Reply:
Whoops! I interverted `sqrt` and `log`.
```suggestion
`{1, ..., n}` of size `n / exp (O (sqrt (log n)))` which does not contain arithmetic progressions of
```